packages/playground/src/weblets/tf_discourse.vue

```diff
@@ -121,7 +127,7 @@ const profileManager = useProfileManager();
 const name = ref(generateName({ prefix: "dc" }));
 const email = ref(profileManager.profile?.email || "");
 const solution = ref() as Ref<SolutionFlavor>;
-const ipv4 = ref(false);
+const ipv4 = ref(true);
```

Comment:
does updating ipv4 to true by default has no effect on other solution?

Reply:
no, it's only for discourse